service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrder.java

```diff
@@ -3,17 +3,20 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.Builder;
 import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
 import uk.gov.hmcts.reform.fpl.model.common.Recital;
 import uk.gov.hmcts.reform.fpl.model.common.Schedule;
 
+import java.util.List;
 import java.util.UUID;
 
 @Data
 @Builder(toBuilder = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class CaseManagementOrder {
     private final String hearingDate;
-    private final UUID hearingDateId;
+    private final String hearingDateId;
```

Comment:
why back to string?

Reply:
Bad merge conflict resolution, reverting.